content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-client-extensions-model.mdx

```diff
@@ -0,0 +1,135 @@
+---
```

Comment:
I think the URL of this would be much better if the files was `100-model.mdx`: `.../prisma-client/client-extensions/model`

Reply:
Good point - I'll change the filenames